src/coreclr/vm/field.h

```diff
@@ -324,6 +324,9 @@ class FieldDesc
     // Return -1 if the type isn't loaded yet (i.e. if LookupFieldTypeHandle() would return null)
     UINT GetSize();
 
+    // Return -1 if the type is a structure and pMTOfValueTypeField is NULL and LookupFieldTypeHandle() returns null
```

Comment:
Do we actually depend on this returning -1 anywhere? It seems we always assume that it is going to succeed. Should we make this more restrictive and assert instead of returning -1?